Client.Core/Flux/Domain/FluxRecord.cs

```diff
@@ -23,6 +23,23 @@ public class FluxRecord
         /// </summary>
         public Dictionary<string, object> Values { get; } = new Dictionary<string, object>();
 
+        /// <summary>
+        /// The record's rows.
+        /// </summary>
+        public List<Row> Rows { get; } = new List<Row>();
```

Comment:
Please use simple `object` instead of `Row`:

```suggestion
        public List<object> Row { get; } = new List<object>();
```